diarize.py

```diff
@@ -71,10 +84,11 @@
     whisper_results.append(segment._asdict())
 # clear gpu vram
 del whisper_model
-torch.cuda.empty_cache()
+if args.device == "cuda":
+    torch.cuda.empty_cache()
```

Comment:
no need for this as `torch.cuda.empty_cache()` works whether a gpu is available or not